testing/model-mesh/openvino-serving-runtime.yaml

```
@@ -0,0 +1,63 @@
+# Copyright 2022 IBM Corporation
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+apiVersion: serving.kserve.io/v1alpha1
+kind: ServingRuntime
+metadata:
+  annotations:
+    create-route: "True"
+    enable-auth: "True"
+  name: ovms-1.x
+  labels:
+    name: modelmesh-serving-ovms-1.x-SR
+spec:
+  supportedModelFormats:
+    - name: openvino_ir
+      version: opset1
+      autoSelect: true
+    - name: onnx
+      version: "1"
+      autoSelect: true
+
+  protocolVersions:
+    - grpc-v1
+  multiModel: true
+
+  grpcEndpoint: "port:8085"
+  grpcDataEndpoint: "port:8001"
+
+  containers:
+    - name: ovms
+      image: openvino/model_server:2022.2
```

```suggestion
      image: docker.io/openvino/model_server:2022.2
```
better with a fullly qualified image address

---

100% agreed, but I won't make the change here.